common/types/query.go

```diff
@@ -328,3 +328,30 @@ func (param *MarketsQuery) Check() error {
 	}
 	return nil
 }
+
+// OrdersQuery definition
+type OrdersQuery struct {
+	Offset *uint32 `json:"offset,omitempty,string"` //Option
+	Limit  *uint32 `json:"limit,omitempty,string"`  //Option
```

Comment:
Why do you use pointer?

Reply:
The field is optional, use can not assign them so that will use default by backend service.